tools/generate_password_blacklist.php

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env php
```

Comment:
dont need this

Reply:
Are you opposed to it? In general I like being able to run the script just using `./script`.

It also makes it so that it prints the below comments to the terminal when you run it.

Reply:
no just curious why

Reply:
I dunno, just a small thing. Less to type.

Reply:
Like this it will direct the server to use the first occurrence of `php` in the $PATH of the user running the script.
which might be dangerous if someone have access to the server (well, in that case, it might be to late anyway).

not using it, but just running the script from command line with `php tools/generate_password_blacklist.php` will use the first occurrence of `php` in the $PATH of the user running the script. So the same thing.

This line is thus completely non necessary. It is best not to include it, this way if the script as the `execute` bit set for the user(or group) the person would have to run it with `php tools/generate_password_blacklist.php`  instead of just typing `tools/generate_password_blacklist.php`